### PR TITLE
Docs/new test cases

### PR DESCRIPTION
**Pull request by jb-adams**
Adjusted the compliance docs to reflect changes to the API schema/new test cases.

### CODE REVIEW

**Comment by jb-adams** [comment]
Ran the compliance against the 3 implementations, and all tests passed on each.

**Comment by jb-adams** [comment]
Fixed 2 Broken links to API specification, @emi80 and @saupchurch can you review?

File: docs/source/data/continuous_ticket.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "id": "5e22e009f41fc53cbea094a41de8798f",
   "version": "1.0",
   "studyID": "f3ba0b59bed0fa2f1030e7cb508324d1",
   "url": "https://url/to/continuous/file",
+  "units": "count",
   "tags": [
```
Comment (saupchurch) [inline docs/source/data/continuous_ticket.json]: "tags" has been removed from spec

File: docs/source/data/expression_ticket.json [hunk 2]
```diff
@@ -1,11 +1,10 @@
 {
-  "id": "ac3e9279efd02f1c98de4ed3d335b98e",
   "version": "1.0",
   "studyID": "f3ba0b59bed0fa2f1030e7cb508324d1",
   "url": "https://url/to/expression/file",
+  "units": "TPM",
   "tags": [
```
Comment (saupchurch) [inline docs/source/data/expression_ticket.json]: "tags" has been removed from spec

File: docs/source/data/continuous_ticket.json (hunk 1 above)
Comment (jb-adams) [reply docs/source/data/continuous_ticket.json]: Removed "tags" from this model

File: docs/source/data/expression_ticket.json (hunk 2 above)
Comment (jb-adams) [reply docs/source/data/expression_ticket.json]: removed "tags" from this model

**Comment by jb-adams** [comment]
@saupchurch can I merge?

**Pull request merged**